app/models/batch_context.rb

```diff
@@ -31,7 +34,6 @@ class BatchContext < ApplicationRecord
   validate :verify_staging_location_path
   validate :verify_file_manifest_exists, if: :using_file_manifest
   validate :verify_output_dir_no_exists, unless: proc { persisted? }
-  validate :verify_processing_configuration_for_ocr, if: proc { Settings.ocr.enabled && manually_corrected_ocr == true }
 
```

Comment:
this is not relevant anymore, there is a new radio button for indicating you have OCR, and you can't select the manually created OCR option unless you select it